apps/studio/src/components/sidebar/core/DatabaseDropdown.vue

```diff
@@ -5,12 +5,21 @@
       <a class="refresh" @click.prevent="refreshDatabases" :title="'Refresh Databases'">
         <i class="material-icons">refresh</i>
       </a>
+      <a class="refresh" @click.prevent="$modal.show('config-add-database')" :title="'Add Database'">
+        <i class="material-icons">add</i>
+      </a>
     </div>
+    <modal class="vue-dialog beekeeper-modal save-add-database" name="config-add-database" height="auto" :scrollable="true">
+      <div class="dialog-content">
+        <add-database-form :connection="connection" @databaseCreated="databaseCreated" @cancel="$modal.hide('config-add-database')"></add-database-form>
```

Comment:
So if you click this, fill it out, then cancel, then go to create a new DB again it will be partially filled out. Is that right? If not we should add a v-if for whether the modal is open. It will also load all charsets on first render